app/assets/stylesheets/application.scss

```diff
@@ -192,11 +198,11 @@ hr {
   border-radius: 10px; /* Rounded corners */
   font-size: 10px; /* Increase font size */
   font-family: 'Rubik',sans-serif;
-}
+};
 
 #myBtn:hover {
   background-color: #555; /* Add a dark-grey background on hover */
-}
+};
 
 // .footer {
 //   padding-top: 80px;
```

Comment:
Can these comments be removed since they aren't references to what's happening?

Reply:
We will leave these here until we are done done.